src/Microsoft.AspNet.Mvc.Core/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -25,7 +26,7 @@ public class DefaultHtmlGenerator : IHtmlGenerator
             typeof(DefaultHtmlGenerator).GetTypeInfo().GetDeclaredMethod(nameof(ConvertEnumFromString));
 
         private readonly AntiForgery _antiForgery;
-        private readonly IScopedInstance<ActionBindingContext> _bindingContextAccessor;
+        private readonly IClientModelValidatorProvider _clientModelValidatorProvider;
```

Comment:
@pranavkm is doing another change to get rid of these types of facades. I think we're ok to just use options directly here and not bother with the composite.